packages/wallet/ui/src/components/ProvisionDialog.jsx

```diff
@@ -60,6 +71,18 @@ export const useProvisionPoolMetrics = (unserializer, leader) => {
   return data;
 };
 
+/**
+ *
+ * @param {ProvisionPoolMetrics} provisionPoolData
```

Comment:
non-blocking: the current factoring is really:
```suggestion
 * @param {ProvisionPoolMetrics=} provisionPoolData
```
It would be tighter to make this take a defined type and put the lax condition at the call site,
```
const provisionPoolLow = provisionPoolData && isProvisionPoolLow(provisionPoolData);
```

Reply:
Thanks, just added `ts-check` and fixed